emsdk.py

```diff
@@ -1014,15 +1013,48 @@ def xcode_sdk_version():
     return subprocess.checkplatform.mac_ver()[0].split('.')
 
 
+def cmake_target_platform_no_tool_arch(tool):
+    if ARCH == 'aarch64':
+      return 'ARM64'
+    else:
+      return 'x64' if tool.bitness == 64 else 'Win32'
```

Comment:
Is the return value here supposed to be uppercase?

What happens when this function is called on non-windows platforms?

Reply:
According to [https://cmake.org/cmake/help/latest/generator/Visual%20Studio%2017%202022.html#platform-selection](https://cmake.org/cmake/help/latest/generator/Visual%20Studio%2017%202022.html#platform-selection) yes. This method should not be called on non-Windows platforms as it is called only when generator is "Visual Studio".